runtime/doc/lsp.txt

```diff
@@ -1611,7 +1611,8 @@ get({filter})                                       *vim.lsp.inlay_hint.get()*
         local hint = vim.lsp.inlay_hint.get({ bufnr = 0 })[1] -- 0 for current buffer
 
         local client = vim.lsp.get_client_by_id(hint.client_id)
-        resolved_hint = client.request_sync('inlayHint/resolve', hint.inlay_hint, 100, 0).result
+        local resp = client.request_sync('inlayHint/resolve', hint.inlay_hint, 100, 0)
+        local resolved_hint = assert(resp and resp.result, resp.err)
```

Comment:
I don't think that's appropriate here. I added the assert() to avoid confusion when trying out this code. But we need to avoid the urge to explain basic Lua concepts in every code example, that makes the examples very verbose.

Reply:
Fair enough.